feign-hystrix-opentracing/src/main/java/feign/opentracing/hystrix/TracingConcurrencyStrategy.java

```diff
@@ -70,20 +71,22 @@ private TracingConcurrencyStrategy() {
 
     private class OpenTracingHystrixCallable<S> implements Callable<S> {
         private Callable<S> delegateCallable;
-        private Span parent;
+        private ActiveSpan parentSpan;
 
         public OpenTracingHystrixCallable(Callable<S> delegate) {
             this.delegateCallable = delegate;
-            this.parent = spanManager.current().getSpan();
+            parentSpan = tracer.activeSpan();
         }
 
         @Override
         public S call() throws Exception {
-            SpanManager.ManagedSpan activate = spanManager.activate(parent);
+            ActiveSpan.Continuation continuation = parentSpan != null
+                    ? parentSpan.capture() : NoopActiveSpanSource.NoopContinuation.INSTANCE;
```

Comment:
Does it mean that, when using a valid tracer, there will always be a valid `parentSpan#capture` at this point?